src/components/panels/RemoteAppsPanel/RemoteAppsPanel.tsx

```diff
@@ -0,0 +1,44 @@
+import useModelStatus from "hooks/useModelStatus";
+import { ReactElement } from "react";
+
+import { generatePanelTableRows } from "../shared";
+
+import "../_panels.scss";
+
+type Props = {
+  entity: string;
+};
+
+export default function RemoteAppsPanel({
+  entity: appId,
+}: Props): ReactElement {
+  // Get model status info
+  const modelStatusData = useModelStatus();
+
+  const remoteApp = modelStatusData["remote-applications"][appId];
+  interface tableData {
```

Comment:
Typically you want to collect all of the type definitions in one spot so lets move this to the top with the `Props` type.